mne/utils/check.py

```diff
@@ -859,7 +859,7 @@ def _check_stc_units(stc, threshold=1e-7):  # 100 nAm threshold for warning
              % (1e9 * max_cur))
 
 
-def _check_qt_version(*, return_api=False):
+def _check_qt_version(*, return_api=False, check_usable_display=True):
```

Comment:
I added a new argument to prevent `_check_qt_version` and `_init_mne_qtapp` from calling eachother endlessly. Do you see a better approach?

Reply:
I think this is good!